js/src/experimental/vercel/middleware.ts

```diff
@@ -157,7 +120,8 @@ export function LangSmithMiddleware(config?: {
           },
         }
       );
-      return traceableFunc(params);
+      const res = await traceableFunc(params);
```

Comment:
_q_: is this intended?

Reply:
Yeah seeing if it helps some subtle timing issues